src/components/Consent/LinkConsentDialog.tsx

```diff
@@ -178,6 +178,7 @@ export default function LinkConsentDialog({
       mfa_enabled: authUser.mfa_enabled || false,
       last_login: authUser.last_login || new Date().toISOString(),
       profile_picture_url: authUser.read_profile_picture_url || "",
+      deleted: authUser.deleted || false,
```

Comment:
do we really need to store these information inside consent? 🤔

Reply:
verified_by is currently using userbase on FE, but looking at BE, we can switch the type to string and use user's external id.